java/code/webapp/WEB-INF/pages/ssm/ssmindex.jsp

```diff
@@ -173,6 +173,7 @@
                         <rhn:require acl="all_systems_in_set_have_feature(ftr_package_refresh)">
                             <li><bean:message key="ssm.overview.misc.packageprofiles"/></li>
                         </rhn:require>
+                        <li><bean:message key="ssm.overview.schedule.assign"/></li>
```

Comment:
Just  to make sure: no restriction for all the type of user right?

Reply:
IIUC, yes. But as the authors of the RFC, maybe @mcalmer or @hustodemon have some other opinion?